setup.py

```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```

Comment:
Surely this should remain at 1.7... otherwise people still using 1.7 [like the person who linked me to this PR] will get failures.